doc/ReleaseNotes/_ReleaseNotes.md

```diff
@@ -63,6 +63,7 @@
 * Add support for Flyout anchor
 * Improved XAML designer support
 * Improved DependencyObject performance under AOT (JS dynCalls for overrides/delegates inside of EH blocks)
+* Add support for MatrixTransform, UIElement.TransformToVisual now returns a MatrixTransform
```

Comment:
This should be documented as a breaking change, I think...

Reply:
Is sounds that it will conflict with my changes for the transforms ^^

Reply:
Could be yes, I'll move it.